README.md

```diff
@@ -1 +1,22 @@
-# ArtCourses-NAN
\ No newline at end of file
+## ArtCourses-NAN ##
+
+team mates :   
+  * @AhlamKadour  
+  * @NaremanFero  
+  * @Noorb7188
+
+we are planning of building an app about Art courses and in the app we are enabling the user to choose a course to participate in.  
+
+steps :
+ - we will build our database using `build.sql`.
+ - the user must sign up to be able to log in, which then we will add him to our users table in the database, that this table has { user_id, user_name, user_email, user_password } columns.
+ - we will use `bcrypt` to save a hashed password in the databse.
+- we will use cookies to authenticate the user's log in to let him see the data about the courses.
```

Comment:
In lline 14 : 
we use cookies to track the user behavior in the website, not to authenticate the user's log in :sweat_smile: